rpcs3/Emu/Cell/PPUFunction.cpp

```diff
@@ -1771,7 +1771,7 @@ extern std::string ppu_get_function_name(const std::string& _module, u32 fnid)
 	case 0xe0da8efd: return "sys_spu_image_close";
 	case 0xe66bac36: return "console_putc";
 	case 0xe6f2c1e7: return "sys_process_exit";
-	case 0xe76964f5: return "sys_game_board_storage_read";
+	case 0xe76964f5: return "_sys_game_board_storage_read";
```

Comment:
Ok maybe there are both syscall and firmware function, in this case add an underscore to the syscall name and keep the old function.

Reply:
Also need to add `{ "sys_game_board_storage_read", reinterpret_cast<u64>(ppu_execute_syscall) },` in   https://github.com/RPCS3/rpcs3/blob/64579eef7eac1e8ac1a7195878368bb58b9bef81/rpcs3/Emu/Cell/PPUThread.cpp#L3283 for backwards compatibilty.

Reply:
OK, added.

Reply:
> Ok maybe there are both syscall and firmware function, in this case add an underscore to the syscall name and keep the old function.

Referring to another syscall "sys_net_eurus_post_command", they both:
1. are registered in rpcs3/Emu/Cell/lv2/lv2.cpp with `BIND_SYSC()`
2. are implemented in rpcs3/Emu/Cell/lv2/sys_XXX.cpp
3. appear in rpcs3/Emu/Cell/PPUFunction.cpp with an underscore before their names.

Reply:
These names are special and are the origin of the hexadecimal constants on the left, not the other way around.

Reply:
Excepting for reverting the syscall name in PPUFunction.cpp, what else should also be changed?
Thank you for your advice!

Reply:
Updated the syscall names.